src/app/tabs/tabs.component.ts

```diff
@@ -1,11 +1,34 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { TabsService } from './tabs.service';
+import { BrowserStorageService } from '@services/storage.service';
 
 @Component({
   selector: 'app-tabs',
   templateUrl: 'tabs.component.html',
   styleUrls: ['tabs.component.scss']
 })
-export class TabsComponent {
-	showTeam = false;
-	showReview = true;
+export class TabsComponent implements OnInit {
+	showReview = false;
+  showChat = true;
+  noOfTodoItems = 0;
+  noOfChats = 0;
+
+  constructor (
+    private tabsService: TabsService,
+    private storage: BrowserStorageService
+  ) {}
+
+  ngOnInit() {
```

Comment:
@shawnm0705 ngOnInit() will only initiated once during class instantiation. And i saw the `noOfTodoItems` code placed inside ngOnInit, so if updating the number of item/chats once per app load is intended? If they aren't you can consider using ionic's `ionViewWillEnter` lifecycle hook